pkg/v1/providers/yttcb/clusterbootstrap.yaml

```diff
@@ -16,18 +16,73 @@
 #@   end
 #@ end
 
+#@ def split_comma_values(value):
+#@  return value.split(",") if value else []
+#@ end
+
+#@ def is_dualstack():
+#@   return data.values.TKG_IP_FAMILY in ["ipv4,ipv6", "ipv6,ipv4"]
+#@ end
+
+#@ def get_ipv4_service_cidr():
+#@   if not is_dualstack():
+#@     return data.values.SERVICE_CIDR
+#@   end
+#@   if data.values.TKG_IP_FAMILY.startswith("ipv4"):
+#@     return data.values.SERVICE_CIDR.split(",")[0]
+#@   end
+#@   if data.values.TKG_IP_FAMILY.startswith("ipv6"):
+#@     return data.values.SERVICE_CIDR.split(",")[1]
+#@   end
+#@ end
+
+#@ def get_ipv6_service_cidr():
+#@   if not is_dualstack():
+#@     return data.values.SERVICE_CIDR
+#@   end
+#@   if data.values.TKG_IP_FAMILY.startswith("ipv6"):
+#@     return data.values.SERVICE_CIDR.split(",")[0]
+#@   end
+#@   if data.values.TKG_IP_FAMILY.startswith("ipv4"):
+#@     return data.values.SERVICE_CIDR.split(",")[1]
+#@   end
+#@ end
+
 #@ def antrea_configs_exist():
 #@   return (data.values.NSXT_POD_ROUTING_ENABLED or
 #@           data.values.ANTREA_NO_SNAT or
+#@           data.values.ANTREA_DISABLE_UDP_TUNNEL_OFFLOAD or
```

Comment:
these could be reduced only for featureflags or a simpler value check from data like `cni: antrea`